### PR TITLE
Auto-Accept Marketplace terms

### LINKED ISSUE #182: New-CmAzIaasVm - Automate accepting of VM marketplace terms
If plan settings are provided in order to provision a custom marketplace image, call Set-AzMarketplaceTerms to automatically accept the terms and conditions of the custom image, output a warning highlighting that terms will be automatically accepted.

### PR DESCRIPTION
**Pull request by deepend-dev**
closes #182

### CODE REVIEW

**Review by tm-luke — COMMENTED** [review]
Can you run this in the integration pipeline?

File: Cloudmarque.Azure/Resources/Project/networking/csv/vnet.csv [hunk 1]
```diff
@@ -17,4 +17,4 @@ Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1
 Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1-Snet-fwha1,10.0.0.80/28,Iaas-Test-Hub2-Vnet1-Snet-Nsg-fwha1,Iaas-Test-Hub2-Vnet1-Snet-Udr-fwha1,,
 Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1-Snet-management1,10.0.0.96/27,Iaas-Test-Hub2-Vnet1-Snet-Nsg-management1,Iaas-Test-Hub2-Vnet1-Snet-Udr-management1,,
 Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1-Snet-fwpub1,10.0.0.128/27,Iaas-Test-Hub2-Vnet1-Snet-Nsg-fwpub1,Iaas-Test-Hub2-Vnet1-Snet-Udr-fwpub1,,
-Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1-Snet-dc1,10.0.0.192/27,Iaas-Test-Hub2-Vnet1-Snet-Nsg-dc1,Iaas-Test-Hub2-Vnet1-Snet-Udr-dc1,,
\ No newline at end of file
+Iaas-Test-Hub2-Rg,UKWest,Iaas-Test-Hub2-Vnet1,10.0.0.0/23,,,Iaas-Test-Hub2-Vnet1-Snet-dc1,10.0.0.192/27,,,,
```
Comment (tm-luke) [inline Cloudmarque.Azure/Resources/Project/networking/csv/vnet.csv:20]: Why are we removing this?

File: Cloudmarque.Azure/Resources/Schema/New-CmAzIaasVm.Schema.json [hunk 2]
```diff
@@ -302,7 +302,7 @@
                                                             "type": "string",
                                                             "title": "OSType",
                                                             "description": "The over arching type of OS to use.",
-                                                            "enum": ["Windows", "Linux"]
+                                                            "enum": ["Windows", "Linux", "windows", "linux"]
```
Comment (tm-luke) [inline Cloudmarque.Azure/Resources/Schema/New-CmAzIaasVm.Schema.json:305]: This shouldn't be needed

**Review by tm-luke — CHANGES_REQUESTED** [review]

File: Cloudmarque.Azure/Resources/Schema/New-CmAzIaasVm.Schema.json (hunk 2 above)
Comment (deepend-dev) [reply Cloudmarque.Azure/Resources/Schema/New-CmAzIaasVm.Schema.json:305]: It seems to be case sensitive.

File: Cloudmarque.Azure/Resources/Project/networking/csv/vnet.csv (hunk 1 above)
Comment (deepend-dev) [reply Cloudmarque.Azure/Resources/Project/networking/csv/vnet.csv:20]: to get one subnet access internet so that omsagnt can install

**Review by tm-luke — APPROVED** [review]

**Pull request merged**